crates/config/README.md

```diff
@@ -86,8 +86,8 @@ gas_reports_ignore = []
 # solc = '0.8.10'
 auto_detect_solc = true
 offline = false
-optimizer = true
-optimizer_runs = 200
+optimizer = false
+optimizer_runs = 0
```

Comment:
I think it makes sense to still default to `optimizer_runs = 200` given that Hardhat also does this and it is a reasonable default if one turns the optimizer on